src/environments/environment.ts

```diff
@@ -5,7 +5,7 @@
 /** @ignore */
 export const environment = {
    production: false,
-   cms_url: 'http://localhost:9009/CMSforce/as'
+   cms_url: 'http://localhost:9002'
```

Comment:
Local requests should go through Zuul too

Reply:
this should be fixed now.